assets/src/components/TableAssignment.js

```diff
@@ -85,40 +105,54 @@ function CustomAssignmentTable (props) {
           {data.map((row, i) => {
             return (
               <TableRow key={i}>
-                {row.map((prop, j) => {
-                  let displayProp = true
-                  let displayBorder = true
-                  if (data[i - 1]) {
-                    // first item in the row the logic handles display of week property
-                    if (j == 0 && data[i][0] === data[i - 1][0]) {
-                      displayProp = false
+                {
```

Comment:
This commit is 
1. reducing the size of the table, making it scrollable
2. current week indicator with the color change

Reply:
I like the current week indicator!

A question about making the table smaller - why set a fixed height for the table at `400px`? Especially on mobile devices, nearly half of the bottom of the screen is blank, and users are forced to scroll.

Reply:
The idea with assignment table design is when the user loads the page the table will be auto-scrolled to the `current_week` and that way student is guided what needs to be done for that week in the course. So that's why we had to make the table size smaller. So if they want to see a previous assignment they can scroll and look around.

The Week number is aligned with the start of the course term. So this makes them easier thinking for assignment/quiz preparation how far along they are in a term.

I know that leave the rest of the screen is blank for mobile. It has a better visual from the landscape mode.  

I was playing around with various devices from the Chrome Dev tool. I noticed for tablets the screen is blank from the bottom and left side as well.

is there a better way we can design this?

Reply:
> I noticed for tablets the screen is blank from the bottom and left side as well.

If you refresh after you open Chrome dev tools in the tablet/phone emulator, this blank spacing should go away.

I'm not sure what the best way to design this is. Perhaps having the table take up more space and highlighting the current week is sufficient.

Reply:
auto-scroll feature to the current week is a feature that asked for, I don't know if we let the table take the rest of the space. I will work auto-scroll tomorrow and see the feasibility of it. I see scroll to row is supported as part of [react-virtualization](https://material-ui.com/demos/tables/#virtualized-table) table. It would be a change to the existing setup which is using simple material-UI table.

Reply:
Okay sounds good!